package.json

```diff
@@ -22,7 +22,8 @@
     "request-promise": "^4.1.1",
     "ts-node": "^3.1.0",
     "tslint": "^5.4.3",
-    "typescript": "^2.4.1"
+    "typescript": "^2.4.1",
+    "strong-docs": "git://github.com/strongloop/strong-docs.git#master"
```

Comment:
Yes, Thats a TODO. I am still fixing sdocs issue as and how they come to support TS. Ones it is in fairly good shape and well tested for TS, we will publish it, and use the version here, instead of github url.